### PR TITLE
Update docs with Ubuntu 20.04

### PR DESCRIPTION
**Pull request by CyanDevs**
Signed-off-by: Chris Yan <chrisyan@microsoft.com>

### CODE REVIEW

**Comment by CyanDevs** [comment]
bors try

File: docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+sudo ./sgx_linux_x64_driver.bin
+```
+
+> This may not be the latest Intel SGX DCAP driver.
+> Please check with [Intel's SGX site](https://01.org/intel-software-guard-extensions/downloads)
+> if a more recent SGX DCAP driver exists.
+
+### 3. Install the Intel and Open Enclave packages and dependencies
+```bash
+sudo apt -y install clang-8 libssl-dev gdb libsgx-enclave-common libsgx-quote-ex libprotobuf10 libsgx-dcap-ql libsgx-dcap-ql-dev az-dcap-client open-enclave
```
Comment (CyanDevs) [inline docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md:51]: To change to clang-10 when https://github.com/openenclave/openenclave/pull/4051 is merged

Comment (CyanDevs) [inline docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md:51]: libprotobuf10 is deprecated, use libprotobuf17

File: docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+If you wish to use the Ninja build system rather than make, also install
+```bash
+sudo apt -y install ninja-build
+```
+
+If you wish to make use of the Open Enclave CMake package, please install CMake:
+
+```
+sudo apt-get install python-pip
```
Comment (CyanDevs) [inline docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md:66]: python-pip package not available?

File: docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md (hunk 1 above)
Comment (DylanWangWQF) [reply docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md:51]: Hi, @CyanDevs . On Ubuntu 20.04, added the dependency "`https://packages.microsoft.com/ubuntu/20.04/prod"`. 
But when run `sudo apt -y install open-enclave`, cannot locate open-enclave. Currently, it's not updated?

Comment (CyanDevs) [reply docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md:51]: Hi @DylanWangWQF, the Ubuntu 20.04 package is not available yet, so this step will not work. Keep an eye out for future releases to see when Ubuntu 20.04 is supported :)

File: docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md (hunk 2 above)
Comment (CyanDevs) [reply docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md]: Used for installing cmake/ansible mainly. Switched to use python3-pip as our Ansible installation script has been using that.

File: docs/GettingStartedDocs/Contributors/NonAccMachineSGXLinuxGettingStarted.md [hunk 3]
```diff
@@ -52,22 +52,31 @@ NOTE: In case the Intel SGX APT source repository is not added to your system. R
...
 wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
 ```
 
-Update the apt
+Update apt package database
 ```bash
 sudo apt-get update
 ```
 
 - Or Install Intel DCAP Quote Provider with dpkg manually
```
Comment (CyanDevs) [inline docs/GettingStartedDocs/Contributors/NonAccMachineSGXLinuxGettingStarted.md:73]: Bullet pointing this seems awkward as it is indented compared to the paragraph that follows it

File: docs/GettingStartedDocs/VisualStudioLinux.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ but a non-SGX machine can still be used:
 
 On the Linux build machine, or after opening an ssh session into the VM:
 
-- Install the Open Enclave SDK.  See [installation instructions for Ubuntu 18.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md), [SGX1 instructions](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md).
+- Install the Open Enclave SDK.  See [installation instructions for Ubuntu 18.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md) or [installation instructions for Ubuntu 20.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md) , [SGX1 instructions](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md).
```
Comment (CyanDevs) [inline docs/GettingStartedDocs/VisualStudioLinux.md:39]: This seems to originally have meant 1 or 2, so the change should be 1 or 2 or 3. Changing this to a list would make it easier to read and notice that.

**Review by rs-- — APPROVED** [review]

File: docs/GettingStartedDocs/install_host_verify_Ubuntu_20.04.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Install the Open Enclave Host-Verify SDK (Ubuntu 20.04)
```
Comment (radhikaj) [inline docs/GettingStartedDocs/install_host_verify_Ubuntu_20.04.md:1]: Add WIP- coming soon to the title

File: docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
+# Install the Open Enclave SDK (Ubuntu 20.04)
```
Comment (radhikaj) [inline docs/GettingStartedDocs/install_oe_sdk-Ubuntu_20.04.md:1]: Add WIP- coming soon

**Review by radhikaj — APPROVED** [review]

**Review by radhikaj — APPROVED** [review]
Bors r+

**Pull request merged**